app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx

```diff
@@ -4,6 +4,8 @@ import { FormattedMessage as T } from "react-intl";
 import { useTicketsList } from "./hooks";
 import * as txTypes from "constants/Decrediton";
 
+const QRCode = require("qrcode");
```

Comment:
This is the syntax to import a module in a nodejs app, we want to use ES6/ES7 syntax:

Note the import statement and the usage of promises instead of callback.
```
import QRCode from 'qrcode'
 
// With promises
QRCode.toDataURL('I am a pony!')
  .then(url => {
    console.log(url)
  })
  .catch(err => {
    console.error(err)
  })
 
// With async/await
const generateQR = async text => {
  try {
    console.log(await QRCode.toDataURL(text))
  } catch (err) {
    console.error(err)
  }
}
```

ref: https://www.npmjs.com/package/qrcode#es6es7